script.js

```diff
@@ -268,3 +268,21 @@ function toggle() {
 }
 toggle();
 popup();
+
+// Form Validation
+
+function validation() {
```

Comment:
[OPTIONAL] Kindly consider using ES6 arrow function as it would help you in React module and industry.